extensions/amp-story/0.1/amp-story-share.css

```diff
@@ -43,6 +43,8 @@
   padding: 0 8px !important;
   margin: 0 !important;
   display: flex !important;
+  flex-wrap: wrap !important;
```

Comment:
How is this not applied to the bookend as well?

Reply:
This is now the default behavior, that's overridden by the ``.i-amphtml-story-share-widget-scrollable`` class right after, setting it to ``nowrap`` again